package.json

```diff
@@ -20,6 +20,10 @@
     "@ethersproject/strings": "5.7.0",
     "@ethersproject/transactions": "5.7.0",
     "@solana/web3.js": "1.87.6",
+    "@stablelib/blake2b": "1.0.1",
+    "@taquito/local-forging": "17.4.0",
```

Comment:
don't think we need this anymore.

Reply:
We do if we support hash signing (and depending on your answer for

https://github.com/dfns/dfns-sdk-ts/pull/2#discussion_r1416966861

Reply:
yeah I'm referring to `local-forging` seems like no longer a dep we need.